src/pages/signin/TermsAndLicenses/TermsWithLicenses.js

```diff
@@ -4,35 +4,41 @@ import styles from '../../../styles/styles';
 import CONST from '../../../CONST';
 import TextLink from '../../../components/TextLink';
 import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import LogoWordmark from '../../../../assets/images/expensify-wordmark.svg';
 
 const TermsWithLicenses = ({translate}) => (
-    <View style={[styles.mt6, styles.flexRow, styles.flexWrap]}>
-        <Text style={[styles.loginTermsText]}>
-            {translate('termsOfUse.phrase1')}
-            {' '}
-        </Text>
-        <TextLink style={[styles.loginTermsText]} href={CONST.TERMS_URL}>
-            {translate('termsOfUse.phrase2')}
-        </TextLink>
-        <Text style={[styles.loginTermsText]}>
-            {' '}
-            {translate('termsOfUse.phrase3')}
-            {' '}
-        </Text>
-        <TextLink style={[styles.loginTermsText]} href={CONST.PRIVACY_URL}>
-            {translate('termsOfUse.phrase4')}
-        </TextLink>
-        <Text style={[styles.loginTermsText]}>.</Text>
-        <Text>
+    <View>
+        <View style={[styles.mt1, styles.alignItemsCenter]}>
+            <LogoWordmark height={30} width={80} />
+        </View>
+        <View style={[styles.mt2, styles.flexRow, styles.flexWrap, styles.textAlignCenter]}>
             <Text style={[styles.loginTermsText]}>
-                {translate('termsOfUse.phrase5')}
+                {translate('termsOfUse.phrase1')}
                 {' '}
             </Text>
-            <TextLink style={[styles.loginTermsText]} href={CONST.LICENSES_URL}>
-                {translate('termsOfUse.phrase6')}
+            <TextLink style={[styles.loginTermsText]} href={CONST.TERMS_URL}>
+                {translate('termsOfUse.phrase2')}
+            </TextLink>
+            <Text style={[styles.loginTermsText]}>
+                {' '}
+                {translate('termsOfUse.phrase3')}
+                {' '}
+            </Text>
+            <TextLink style={[styles.loginTermsText]} href={CONST.PRIVACY_URL}>
+                {translate('termsOfUse.phrase4')}
             </TextLink>
             <Text style={[styles.loginTermsText]}>.</Text>
-        </Text>
+            <Text>
+                <Text style={[styles.loginTermsText]}>
+                    {translate('termsOfUse.phrase5')}
```

Comment:
It looks like there should be a `{' '}` above this as well since there is no space between `privacy policy.` and `Money transmission`. Additionally, `licences` is not aligned properly in on iOS.

<img src='https://user-images.githubusercontent.com/3981102/122138110-edcbb300-cdfa-11eb-9346-e0fa196b6696.png' width='400'/>

Reply:
Flex wrap is behaving weirdly in mobile. I tried fixing this but due to the long content size it's getting messed up. Can you pls help me with fixing this.

![Screenshot from 2021-06-16 06-56-16](https://user-images.githubusercontent.com/32242596/122143913-0aee9900-ce70-11eb-900f-e0fd32e26687.png)

![Screenshot from 2021-06-16 06-56-31](https://user-images.githubusercontent.com/32242596/122143910-09bd6c00-ce70-11eb-8cc8-4da168785995.png)

```jsx
import React from 'react';
import {Text, View} from 'react-native';
import styles from '../../../styles/styles';
import CONST from '../../../CONST';
import TextLink from '../../../components/TextLink';
import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
import LogoWordmark from '../../../../assets/images/expensify-wordmark.svg';

const TermsWithLicenses = ({translate}) => (
    <View>
        <View style={[styles.mt1, styles.alignItemsCenter]}>
            <LogoWordmark height={30} width={80} />
        </View>
        <View style={[styles.mt2, styles.flexRow, styles.flexWrap, styles.textAlignCenter]}>
            <Text style={[styles.loginTermsText]}>
                {translate('termsOfUse.phrase1')}
                {' '}
            </Text>
            <TextLink style={[styles.loginTermsText]} href={CONST.TERMS_URL}>
                {translate('termsOfUse.phrase2')}
            </TextLink>
            <Text style={[styles.loginTermsText]}>
                {' '}
                {translate('termsOfUse.phrase3')}
                {' '}
            </Text>
            <TextLink style={[styles.loginTermsText]} href={CONST.PRIVACY_URL}>
                {translate('termsOfUse.phrase4')}
            </TextLink>
            <Text style={[styles.loginTermsText]}>.</Text>
            <Text style={[styles.loginTermsText]}>
                {translate('termsOfUse.phrase5')}
                {' '}
            </Text>
            <TextLink style={[styles.loginTermsText]} href={CONST.LICENSES_URL}>
                {translate('termsOfUse.phrase6')}
            </TextLink>
            <Text style={[styles.loginTermsText]}>.</Text>
        </View>
    </View>
);

TermsWithLicenses.propTypes = {...withLocalizePropTypes};

export default withLocalize(TermsWithLicenses);

```

Reply:
> I tried fixing this but due to the long content size it's getting messed up. Can you pls help me with fixing this.

If you are having trouble with a solution please post in our [public slack channel](https://expensify.slack.com/archives/C01GTK53T8Q) and ask for help. I'm not sure what the issue is here so it is more likely that our community can help you faster than I can alone.

Reply:
Fixed